pkg/virt-launcher/virtwrap/api/converter.go

```diff
@@ -992,6 +990,9 @@ func calculateRequestedVCPUs(vmi *v1.VirtualMachineInstance) uint32 {
 	if vmi.Spec.Domain.CPU != nil && vmi.Spec.Domain.CPU.Cores != 0 {
 		return vmi.Spec.Domain.CPU.Cores
 	}
+	if !vmi.IsCPUDedicated() {
+		return uint32(1)
+	}
 	if cpuRequests, ok := vmi.Spec.Domain.Resources.Requests[k8sv1.ResourceCPU]; ok {
```

Comment:
I believe I missed it, but do we have the check that `uint32(cpuRequests.Value()) != 0`?